src/components/Navigation/index.tsx

```diff
@@ -9,66 +10,71 @@ import { ReactComponent as SandwichMenu } from 'assets/icons/sandwich_menu.svg';
-                    <LogoSvg />
-                </StyledLogoContainer>
-            </NavLink>
-            <StyledNavigationItems>
-                <ul>
-                    <li>
-                        <StyledLink exact activeClassName='active' to='/propose'>
-                            Propose App
-                        </StyledLink>
-                    </li>
-                    <li>
-                        <StyledLink exact activeClassName='active' to='/new-farm'>
-                            New Farm
-                        </StyledLink>
-                    </li>
-                    <li>
-                        <StyledLink exact activeClassName='active' to='/stats'>
-                            Stats
-                        </StyledLink>
-                    </li>
-                    <li>
-                        <StyledLink exact activeClassName='active' to='/my-farm'>
-                            My Farm
-                        </StyledLink>
-                    </li>
-                </ul>
-            </StyledNavigationItems>
-            <Flex align={'center'}>
-                <StyledConnectWalletButton>
-                    Connect
-                    <StyledMetaMaskImageContainer style={{ backgroundImage: `url(${MetaMaskImage})` }} />
-                </StyledConnectWalletButton>
-                <StyledSandwichMenuContainer onClick={() => setSidebar(true)}>
-                    <SandwichMenu />
-                </StyledSandwichMenuContainer>
-            </Flex>
-        </StyledNavigationContainer>
-    )
-}
+const Navigation: React.FC<INavigation> = ({ setSidebar }) => {
+  return (
+    <StyledNavigationContainer
+      css={`
+        background-image: url(${NavigationBarImage});
+      `}
+    >
+      <NavLink exact activeClassName="active" to="/">
+        <StyledLogoContainer>
+          <LogoSvg />
+        </StyledLogoContainer>
+      </NavLink>
+      <StyledNavigationItems>
+        <ul>
+          <li>
+            <StyledLink exact activeClassName="active" to="/propose">
+              Propose App
+            </StyledLink>
+          </li>
+          <li>
+            <StyledLink exact activeClassName="active" to="/new-farm">
+              New Farm
+            </StyledLink>
+          </li>
+          <li>
+            <StyledLink exact activeClassName="active" to="/stats">
+              Stats
+            </StyledLink>
+          </li>
+          <li>
+            <StyledLink exact activeClassName="active" to="/my-farm">
+              My Farm
+            </StyledLink>
+          </li>
+        </ul>
+      </StyledNavigationItems>
+      <Flex align={'center'}>
+        <StyledConnectWalletButton>
+          Connect
+          <StyledMetaMaskImageContainer
+            css={`
+              background-image: url(${MetaMaskImage});
+            `}
+          />
+        </StyledConnectWalletButton>
+        <StyledSandwichMenuContainer onClick={() => setSidebar(true)}>
```

Comment:
Also, just noticed—this is a `div`, not a button—let's change this to be a button and style accordingly. If not, this will not be accessible nor keyboard-focusable.

Reply:
Fixed in c09a156687bc5bea9c84f73335c0c6246df7dfcb